### PR TITLE
Adn/front sentry setup

### PR DESCRIPTION
**Pull request by alexandredamiron**

### CODE REVIEW

File: front/src/index.tsx [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
 import App from './main/App';
+import { persistor, store } from './Store';
+import * as serviceWorker from './serviceWorker';
 import Loader from './common/Loader';
 import { version } from '../package.json';
 
 // Must be required and not imported to be included in production build (strange bug ?)
 require('@sncf/bootstrap-sncf.metier.reseau');
 
+if (SENTRY_CONFIG.react_sentry_dsn) {
```
Comment (flomonster) [inline front/src/index.tsx:25]: If it's an empty string does it returns false?

File: front/src/applications/stdcm/views/OSRDSTDCM.js [hunk 2]
```diff
@@ -17,6 +17,24 @@ export const stdcmRequestStatus = {
...
+  keylog.push(keypressed);
+  if (keylog.join('') === 'boum') {
+    const ascii = [
+      '',
+      ' ___________________    . , ',
+      '(___________________|~~~~~X.;',
+      '                     ` `"  ;',
+      '            TNT',
+    ].join('\n');
+    throw ascii;
```
Comment (flomonster) [inline front/src/applications/stdcm/views/OSRDSTDCM.js:32]: I don't really understand what it's for

File: front/launch_dev_front.sh [hunk 3]
```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```
Comment (anisometropie) [inline front/launch_dev_front.sh:1]: why did you delete that script ?

File: front/src/index.tsx (hunk 1 above)
Comment (anisometropie) [reply front/src/index.tsx:25]: empty strings evaluate as falsy and we do not enter the if block

File: front/src/applications/stdcm/views/OSRDSTDCM.js (hunk 2 above)
Comment (alexandredamiron) [reply front/src/applications/stdcm/views/OSRDSTDCM.js:32]: It is for testing errors recording. Type 'boum' and see if you see that exception in sentry. It does not stops execution

File: front/src/index.tsx (hunk 1 above)
Comment (alexandredamiron) [reply front/src/index.tsx:25]: not in javascript @flomonster , see above and https://developer.mozilla.org/en-US/docs/Glossary/Truthy

File: front/launch_dev_front.sh (hunk 3 above)
Comment (alexandredamiron) [reply front/launch_dev_front.sh:1]: @flomonster told me it is not useful anymore

File: front/src/applications/stdcm/views/OSRDSTDCM.js (hunk 2 above)
Comment (flomonster) [reply front/src/applications/stdcm/views/OSRDSTDCM.js:32]: Ok I see.
